sdk/lib/arbiter/arbiter.h

```diff
@@ -0,0 +1,255 @@
+#ifndef _ARBITER_H
+#define _ARBITER_H
+
+#ifndef _NTOSKRNL_
+#include <ntifs.h>
+#include <wdm.h>
+
+/* C Headers */
+#include <stdio.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdlib.h>
+
+/* PSDK/NDK Headers */
+#define WIN32_NO_STATUS
+#define _INC_WINDOWS
+#define COM_NO_WINDOWS_H
+#define COBJMACROS
+#define CONST_VTABLE
+
+#include <ndk/exfuncs.h>
+#include <ndk/iofuncs.h>
+#include <ndk/kefuncs.h>
+#include <ndk/ldrfuncs.h>
+#include <ndk/mmfuncs.h>
+#include <ndk/obfuncs.h>
+#include <ndk/psfuncs.h>
+#include <ndk/rtlfuncs.h>
+#include <ndk/setypes.h>
+#include <ndk/sefuncs.h>
+#include <ndk/umfuncs.h>
```

Comment:
Since this is the public header for the library, I would suggest only including headers here that are needed to make the definitions below work correctly (which I'm hoping would mean NDK headers aren't needed here at all; and maybe you can get by with just wdm.h instead of ntifs.h).
You could then have another header internal to the library itself that's used as the PCH and includes everything needed for the implementation.

Reply:
Initially, I did not plan to add this. Only after the errors of AppVeyor and Travis did I do this. Therefore, I will try to remove it all. In RoSBE and in MSVS for x86, I have no errors...

Reply:
It looks like a bug when building in RoSBE and in MSVS goes fine, and in AppVeyor and Travis fails. This looks like a different compilation order or a different set of keys to compile. In any case, I expect the identity of the processes for creating ReactOS ...